client/src/Components/Card/index.js

```diff
@@ -36,7 +40,31 @@ export default function CardComponent({ house }) {
     price,
     bdCount,
     baCount,
-  } = house;
+  },
+}) {
+  const [favorite, setFavorite] = useState();
+  const [error, setError] = useState();
+  const [open, setOpen] = useState(false);
+
+  const classes = useStyles();
+
+  const handleClose = (event, reason) => {
+    if (reason === 'clickaway') {
+      return;
+    }
+    setOpen(false);
+  };
+
+  const addToFav = async () => {
+    try {
```

Comment:
before sending another request you need to make clear the state, to make sure that the condition on render work as expected